python/fledge/services/core/server.py

```diff
@@ -278,6 +278,26 @@ class Server:
             'order': '10'
         },
     }
+    _LOGGING_DEFAULT_CONFIG = {
+            'logLevel': {
+                         'description': 'Minimum logging level reported for Core server',
+                         'type': 'enumeration',
+                         'displayName': 'Minimum Log Level',
+                         'options': ['debug', 'info', 'warning', 'error', 'critical'],
```

Comment:
better to keep 
['Debug', 'Info', 'Notice', 'Warning', 'Error', 'Critical']

Titlecase and two more levels. if not all seven.